ansible/wordpress.yml

```diff
@@ -10,6 +10,12 @@
     wp_content_dir: "{{ wp_install_dir }}/wp-content"
   roles:
     - ansible-os-ubuntu-16
+    - role: cristian04.cloudwatch-monitoring
+      cron_specs:
+        - name: "Cloudwatch monitoring"
+          minute: "*/5"
```

Comment:
Is this frequent enough?

Reply:
Anything over 5 min sampling requires detailing monitoring, which incurs costs.

Reply:
Let’s add that as a comment then.